lib/grammers-client/src/types/callback_query.rs

```diff
@@ -77,7 +77,12 @@ impl CallbackQuery {
 
     /// Load the `Message` that contains the pressed inline button.
     pub async fn load_message(&self) -> Result<types::Message, InvocationError> {
-        todo!()
+        Ok(self
+            .client
+            .get_messages_by_id(self.chat(), &[self.query.msg_id])
+            .await?
+            .get(0)
```

Comment:
I think this will need to be a `pop` too if we want the function to return the owned type. When I mentioned `get(0)` I was thinking of user code. It was a brain fart.

Reply:
ok